test/e2e/app-dir/test-template/{{ toFileName name }}/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
This should probably disable type checking during `next build` to not have that overhead: https://nextjs.org/docs/api-reference/next.config.js/ignoring-typescript-errors

Reply:
I disabled it in both app-dir and normal tests.
But just to make sure do we type-check our tests somewhere else?

Reply:
I'm not sure we should disable this by default in new tests as we probably want to ensure any tests written in TypeScript are working as expected 🤔

Reply:
I usually use vscode to show all the warnings, but I just realized that vscode doesn't work well with our tests (It wants `import React from 'react'` for example).
Anyway, I'll remove it and we can add it later. Both are better than the current defaults.